scripts/upgrade-requirements.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/bash
```

Comment:
I have the same comment as [here](https://github.com/RedHatInsights/vulnerability-engine/pull/378#discussion_r302907840). What is purpose of this script?